imsv-docs-astro/src/content/docs/guides/authentication.mdoc

```diff
@@ -3,9 +3,9 @@ title: Authentication
 ---
 
 Depending on the use case and operation, either API key or
-Sign-in-with-ethereum (SIWE) authentication mechanism may be used.
+user session authentication mechanism may be used.
```

Comment:
I'm not sure about calling it `user session authentication`. what are your thoughts @ncjones